ios/RecordingSession.swift

```diff
@@ -192,16 +193,21 @@ class RecordingSession {
    */
   func finish() {
     ReactLogger.log(level: .info, message: "Finishing Recording with AssetWriter status \"\(assetWriter.status.descriptor)\"...")
-
+    if isFinishing {
+      ReactLogger.log(level: .warning, message: "Tried calling finish() twice while AssetWriter is still writing!")
+      return
+    }
```

Comment:
can you add a new line before and after the `if`? Give it room to breathe

Reply:
There was whitespace before, but the linter complained :D

Reply:
```Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)```

Reply:
What happens if you run `yarn check-ios` in the root? I think it is complaining about the line having one whitespace character (`\n `) instead of just the new-line (`\n`).

Reply:
Yeah - that was the reason. Pretty restrictive